spec/gateway/optics_spec.rb

```diff
@@ -0,0 +1,77 @@
+describe Gateway::Optics do
+  subject(:gateway) { described_class.new }
+
+  let(:api_key) { 'foo' }
+  let(:secret_key) { 'bar' }
+  let(:token) { 'eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJmb28iLCJhdWQiOiJodHRwczovL3VhdC5pY2FzZXdvcmsuY29tL3Rva2VuIiwiaWF0IjoxNTY4MjE2MDg2LCJleHAiOjE1ODExNzYwODZ9.0VbK6jGM3Ux8sHq3ekkz-g5xkLxFY4c_6CRVEkA1Sh4' }
+  let(:bearer_token) { SecureRandom.alphanumeric(20) }
+
+  describe '#request_bearer_token' do
+    before do
+      stub_request(:post, 'https://uat.icasework.com/token?db=hmcts').to_return(status: 200, body: { access_token: bearer_token }.to_json)
+    end
+
+    let(:expected_body) do
+      URI.encode_www_form(
+        grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+        assertion: token
+      )
+    end
+
+    it 'sends a request for a token' do
+      gateway.request_bearer_token(jwt_token: token)
+      expect(WebMock) .to have_requested(:post, 'https://uat.icasework.com/token?db=hmcts').with(
```

Comment:
`expect(WebMock) .` there's extra whitespace, not sure how that even works?

Reply:
Yikes, fixed.  Seemed to have no effect on the test.